src/intervention.jl

```diff
@@ -161,3 +161,59 @@ function optimal_parameter_intervention_for_threshold(prob, p1, obs, threshold,
     ss = duration_constraint(optx, [], Val(true))
     Dict(ps .=> optx), ss, ret
 end
+
+"""
+Find `intervention_par` which minimize `intervention_cost`
+but which do not lead to `thresholds` being violated, throughout `prob.tspan`.
+Neither can `intervention_constraints` be violated.
+`intervention_par` influence `prob` through
+`intervention_continuous_events` and `intervention_discrete_events`,
+"""
+function weakest_intervention_not_violating_thresholds(prob,
```

Comment:
Can we clarify in the docstring what the difference between this and `optimal_parameter_intervention_for_threshold` is?